tiledb/tests/test_libtiledb.py

```diff
@@ -1927,15 +1907,7 @@ def test_varlen_wrong_domain(self):
 
     def test_array_varlen_mismatched(self):
         # Test that we raise a TypeError when passing a heterogeneous object array.
-        A = np.array(
-            [
-                b"aa",
-                b"bbb",
-                b"cccc",
-                np.uint64([1, 3, 4]),
-            ],
-            dtype=object,
-        )
+        A = np.array([b"aa", b"bbb", b"cccc", np.uint64([1, 3, 4])], dtype=object)
```

Comment:
(This is what I was talking about w/ black :/ -- this file has been formatted before, no reason for it to change now)

Reply:
Sigh yeah and there seems to be some incongruence between the version of Black being used for GitHub Actions and what I have installed locally. I set it up to always use the latest stable version, so I don't know which one isn't up to date. I also didn't realize how frequently they make changes to this format.

Reply:
Does black have a way to introduce a format file so its consistent in the repo across versions that change the default?

Reply:
Last time I looked it seems they always do it The Correct Way so no configuration ... except that way changes :)

Reply:
Maybe you can tie the call to Black down in a Docker container to freeze it.  Seems like a heavy hammerl though.